helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestAutoRebalance.java

```diff
@@ -164,9 +169,19 @@ public void testAutoRebalance() throws Exception {
     // kill 1 node
     _participants[0].syncStop();
 
-    boolean result = ClusterStateVerifier
-        .verifyByZkCallback(new ExternalViewBalancedVerifier(_gZkClient, CLUSTER_NAME, TEST_DB));
-    Assert.assertTrue(result);
+    ZkHelixClusterVerifier verifierTestDb = new BestPossibleExternalViewVerifier.Builder(CLUSTER_NAME)
+        .setResources(new HashSet<>(Collections.singleton(TEST_DB)))
+        .setZkClient(_gZkClient)
+        .setWaitTillVerify(TestHelper.DEFAULT_REBALANCE_PROCESSING_WAIT_TIME)
+        .build();
+    Assert.assertTrue(verifierTestDb.verifyByPolling());
+
+    ZkHelixClusterVerifier verifierDb2 = new BestPossibleExternalViewVerifier.Builder(CLUSTER_NAME)
```

Comment:
Please change Db2 to a better name.

Reply:
Changed to "verifierClusterDb2" as the purpose it to validate cluster Db2